source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -82,6 +82,9 @@ public PimsAcquisitionFile GetById(long id)
                 .Include(r => r.PimsPropertyAcquisitionFiles)
                     .ThenInclude(rp => rp.Property)
                     .ThenInclude(p => p.DistrictCodeNavigation)
+                .Include(r => r.PimsPropertyAcquisitionFiles)
+                    .ThenInclude(rp => rp.Property)
```

Comment:
These look like a good place for using the property API and concatenate the results. That way we would have a central location for retrieving property data without having to modify multiple quieries

Reply:
I'll log tech debt for that, but I'm not comfortable doing that for this bugfix since this repository was already pulling the property before this bugfix.

Reply:
https://jira.th.gov.bc.ca/browse/PSP-4741